src/web/libdoc/i18n/translations.json

```diff
@@ -56,5 +56,34 @@
     "generatedBy": "Luotu",
     "on": "",
     "chooseLanguage": "Valitse kieli"
+  },
+  "nl": {
+    "code": "nl",
+    "intro": "Introductie",
+    "libVersion": "Bibliotheek versie",
+    "libScope": "Bibliotheek bereik",
+    "importing": "Importeren",
+    "arguments": "Parameters",
+    "doc": "Documentatie",
+    "keywords": "Actiewoorden",
+    "tags": "Labels",
+    "returnType": "Andwoord type",
+    "kwLink": "Actiewoord link",
+    "argName": "Benoemde parameters",
+    "varArgs": "Variabel aantal parameters",
+    "varNamedArgs": "Variable aantal benoemde parameters",
+    "namedOnlyArg": "Alleen benoemde parameters",
+    "posOnlyArg": "Aleen positionele parameters",
+    "defaultTitle": "Standaard waarde welke wordt gebruikt als geen waarde is gegeven",
+    "typeInfoDialog": "Klik om informatie over dit type te zien",
+    "search": "Zoeken",
+    "dataTypes": "Data types",
+    "allowedValues": "Toetestane Waarden",
+    "dictStructure": "Woordenboek Structuur",
```

Comment:
in RIDE heb ik `dict` als jargon laten staan. Dan is het de vraag of structuur nog niets toevoegd in de context. of dat alleen _dict_ volstaat.